src/main/resources/changelog/scripts/v-1.0.0/metadata/functions/create_rebalance_workspace.sql

```diff
@@ -24,7 +24,8 @@ do $do$
                                                         ('resource_type_map'),
                                                         ('resource_context_map'),
                                                         ('generator_manifest'),
-                                                        ('resource_generator_map');
+                                                        ('resource_generator_map'),
+                                                        ('instance_metadata');
```

Comment:
The current implementation of instance_metadata isn't partitioned. I think it would make sense to partition it, but as mentioned in the instance_metadata comment, this would necessitate inverting the foreign key relationship and using the resource_hash key.

Reply:
Done.